src/pe_reports/report_generator.py

```diff
@@ -1,17 +1,38 @@
-"""A tool for creating Posture & Exposure reports."""
+"""A tool for creating Posture & Exposure reports.
+
+Usage:
+    pe-reports REPORT_DATE DATA_DIRECTORY OUTPUT_DIRECTORY
+
+Arguments:
+  REPORT_DATE       Date of the report, format YYYY-MM-DD.
+  DATA_DIRECTORY    The directory where the excel data files are located.
+                    Organized by owner.
+  OUTPUT_DIRECTORY  The directory where the final PDF reports should be saved.
+
+Options:
+  -h --help     Show this message.
+  -v --version  Show version information.
+  --log-level=LEVEL     If specified, then the log level will be set to
+                        the specified value.  Valid values are "debug", "info",
+                        "warning", "error", and "critical". [default: info]
+"""
 
 # Standard Python Libraries
+import logging
 import os
 import sys
```

Comment:
With the removal at the bottom of the file, this import is no longer used.

Reply:
@mcdonnnj, I am not sure I follow. If removed, I begin to get import errors. For now I left "import logging" in so that the code passes pre-commit.

Reply:
@schmelz-ctr The line I am providing feedback on is `import sys`. That is why you are now failing checks.

Reply:
Now included with the addition of logging test. 253b2dd